vendor/github.com/docker/libnetwork/bitseq/sequence.go

```diff
@@ -206,7 +206,7 @@ func (h *Handle) getCopy() *Handle {
 
 // SetAnyInRange atomically sets the first unset bit in the specified range in the sequence and returns the corresponding ordinal
 func (h *Handle) SetAnyInRange(start, end uint64, serial bool) (uint64, error) {
-	if end < start || end >= h.bits {
+	if end < start || end > h.bits {
```

Comment:
Have you update the vendor `libnetwork`?

Reply:
We have vendored the libnetwork is the alibaba `libnetwork`, you can see: `https://github.com/alibaba/libnetwork`, if you want to modify it, you can send PR to our `libnetwork`